numpy/core/tests/test_numeric.py

```diff
@@ -1976,27 +1976,26 @@ def check_function(self, func, fill_value=None):
         fill_kwarg = {}
         if fill_value is not None:
             fill_kwarg = {'fill_value': fill_value}
-        with warnings.catch_warnings():
-            warnings.simplefilter('ignore', DeprecationWarning)
-            for size, ndims, order, dtype in itertools.product(*par):
-                shape = ndims * [size]
 
-                # do not fill void type
-                if fill_kwarg and dtype.str.startswith('|V'):
-                    continue
+        for size, ndims, order, dtype in itertools.product(*par):
```

Comment:
I did not check everything for unnecessary filters, but apparently yes. Unless it would be platform depenended or so, I would assume its just finished.